src/Shared/FrameworkLocationHelper.cs

```diff
@@ -1157,10 +1163,13 @@ private class VisualStudioSpec
             /// </summary>
             public string GetDotNetFrameworkSdkRegistryKey(Version dotNetSdkVersion)
             {
-                string sdkVersionFolder = "4.6"; // Default for back-compat
+                string sdkVersionFolder = "4.7"; // Default for back-compat
```

Comment:
The method is used to generate registryPath. On machine with Dev15.1, the default sdk is 4.7 so very likely 4.7 is the only sdk installed on the machine.